wire-ios/Wire-iOS/Sources/UserInterface/Availability/UserStatusViewController.swift

```diff
@@ -88,4 +90,41 @@ final class AvailabilityTitleViewController: UIViewController {
         feedbackGenerator.prepare()
         feedbackGenerator.impactOccurred()
     }
+
+    // MARK: - Notifications
+
+    private func setupNotificationObservation() {
+        // refresh view when app becomes active
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(updateUserStatusView),
+            name: UIApplication.didBecomeActiveNotification,
+            object: nil
+        )
+
+        // refresh view when some user changes
+        userChangeObservation = userSession.addUserObserver(self, for: user)
+    }
+
+    @objc
+    private func updateUserStatusView() {
+        let view = view as! UserStatusView
```

Comment:
I know this struggle... what one can do create a `var userStatusView: UIView!` (next to `view: UIView!`)and assign the view to both vars. 
Or implement it like this `var userStatusView: UIView { view as! UserStatusView }`.
Even though you have a force cast, it's quite clear that it happens only in one place in the whole view controller.